hue/hue.sh

```diff
@@ -71,7 +71,7 @@ function install_hue_and_configure() {
   install_packages hue || err "Failed to install Hue"
 
   # Stop Hue
-  systemctl stop hue || err "Hue stop action not performed"
+  /etc/init.d/hue force-stop || err "Hue stop action not performed"
```

Comment:
Why do we need this? `stop` does not work?

Reply:
No, it doesn't work...